apps/api/src/server/applications/application/documents/document.js

```diff
@@ -58,8 +58,11 @@ export const buildNsipDocumentPayload = (version) => {
 		originalFilename: version.originalFilename,
 		size: version.size,
 		mime: version.mime,
-		documentURI: buildBlobUri(version.privateBlobContainer, version.privateBlobPath),
-		publishedDocumentURI: buildBlobUri(version.publishedBlobContainer, version.publishedBlobPath),
+		documentURI: buildBlobUri(version.privateBlobContainer ?? '', version.privateBlobPath ?? ''),
```

Comment:
Is it neater to update `buildBlobUri` to accept `string | null`? It seems to already handle it.